templates/docs/examples/patterns/heading-icon/heading-icon-small.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <div class="p-heading-icon--small">
   <div class="p-heading-icon__header">
-    <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/425efe3a-lxd.svg" />
+    <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/425efe3a-lxd.svg" alt="LXD icon"/>
```

Comment:
Example of both problems at once: it’s not representing the LXD icon, it’s representing LXD — and therefore `alt` text is redundant, because it’s immediately followed by the heading “LXD — system containers”.
```suggestion
    <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/425efe3a-lxd.svg" alt=""/>
```